plugins/plugin-iter8/src/components/traffic-split.ts

```diff
@@ -0,0 +1,147 @@
+import { safeLoad, safeDump } from 'js-yaml'
+
+const execSync = require('child_process').execSync
+
+function applyKube(rule) {
+  const yamlRule = safeDump(rule)
+  const command = `cat <<EOF | kubectl apply -f -\n${yamlRule}\nEOF`
+  execSync(command, { encoding: 'utf-8' })
+}
+
+function getLabelInfo(label) {
+  const deployment = execSync(
+    `kubectl get deployments -n ${label['destination_service_namespace']} ${label['destination_workload']} -oyaml`,
+    { encoding: 'utf-8' }
+  )
+  return safeLoad(deployment)['spec']['template']['metadata']['labels']
```

Comment:
This can easily to errors. We should consider using something like [this](https://www.npmjs.com/package/nested-property).